front/src/common/redux/tasks/task.saga.js

```diff
@@ -0,0 +1,45 @@
+import { put, call, all, takeEvery } from "redux-saga/effects";
+import { getTasks, deleteTask, changeTask } from "./task.api";
+import * as types from "./types";
+
+function* getTasksSaga(payload) {
+    try {
+        const response = yield call(getTasks, payload);
+
+        yield put({ type: types.GET_ALL_TASKS_SUCCESS, response });
```

Comment:
Please, provide key `payload` for the data of action

Reply:
done